ee/vellum_ee/workflows/display/nodes/vellum/final_output_node.py

```diff
@@ -45,7 +45,7 @@ def serialize(self, display_context: WorkflowDisplayContext, **_kwargs) -> JsonO
                 "node_input_id": str(node_input.id),
             },
             "inputs": [node_input.dict()],
-            **self.serialize_generic_fields(display_context, exclude=["outputs"]),
+            **self.serialize_generic_fields(display_context),
             "outputs": [
```

Comment:
DevinAI the goal of this refactor is to also remove this key.

If tests fail as a result, let me see, don't iterate on them